utilities/ballistics.py

```diff
@@ -24,14 +25,62 @@ class BallisticsSolution:
 def calculate_ballistics(
     robot_pose: Pose2d, target_position: Translation3d
 ) -> BallisticsSolution:
-    azimuth = math.atan2(
-        target_position.y - robot_pose.y, target_position.x - robot_pose.x
-    )
+    delta = (target_position.y - robot_pose.y, target_position.x - robot_pose.x)
+    distance = math.hypot(delta[0], delta[1])
+    azimuth = math.atan2(delta[0], delta[1])
```

Comment:
nit: for readability
```suggestion
    dy = target_position.y - robot_pose.y
    dx = target_position.x - robot_pose.x
    distance = math.hypot(dy, dx)
    azimuth = math.atan2(dy, dx)
```